doc/UsersGuide/source/AutomatedTesting.rst

```diff
@@ -91,7 +91,7 @@ On supported HPC systems, a :term:`cron job` runs the ``start_rt_auto.sh`` bash
 This script checks the HPC name and sets certain python paths. Then, it runs ``rt_auto.py``, 
 which uses the Github API (through pyGitHub) to check the labels on pull requests to 
 ``ufs-weather-model``. If a PR label matches the HPC name 
-(e.g., hera-intel-RT or cheyenne-gnu-BL), the label provides the HPC  
+(e.g., hera-intel-RT or derecho), the label provides the HPC  
```

Comment:
these are examples of labels, and I recall Derecho supporting GNU, so this should be `derecho-gnu-BL` as a possible label

Reply:
apologies for missing this in my previous reviews.